R/missing_argument_linter.R

```diff
@@ -1,6 +1,7 @@
 #' Missing argument linter
 #'
-#' Check for missing arguments in function calls.
+#' Check for missing arguments in function calls (e.g. `stats::median(1:10, na.rm =)`).
```

Comment:
I would not use a named argument here -- I don't think I saw a single case of this in our code base, whereas there were dozens of usages of the missing argument just being blank (e.g., `stats::median(1:10,)`). So I would put the more common occurrence in the docs.

Reply:
Good point. Using the most common occurrence example now.